src/main/kotlin/com/autonomousapps/tasks/FindDeclarationsTask.kt

```diff
@@ -126,20 +129,20 @@ class LocationMetadata(
   private fun isJavaPlatform(id: String): Boolean = metadata.containsKey(id)
 
   companion object {
-    internal fun of(metadata: Map<String, Boolean>): LocationMetadata = LocationMetadata(metadata)
+    internal fun of(metadata: Map<String, Boolean>): DeclarationMetadata = DeclarationMetadata(metadata)
   }
 }
 
-internal class Locator(private val locationContainer: LocationContainer) {
-  fun locations(): Set<Declaration> {
-    return locationContainer.mapping.asSequence()
-      .filter { (name, _) -> isMain(name) || isAnnotationProcessor(name) }
+internal class Locator(private val declarationContainer: DeclarationContainer) {
+  fun declarations(): Set<Declaration> {
+    return declarationContainer.mapping.asSequence()
+      // .filter { (name, _) -> isForRegularDependency(name) || isForAnnotationProcessor(name) }
```

Comment:
This looks redundant, since the exact same filter is applied when creating the declaration container in the first place.